source/features/easy-toggle-commit-messages.tsx

```diff
@@ -49,7 +62,8 @@ void features.add(import.meta.url, {
 
 Test URLs:
 
-https://github.com/refined-github/sandbox/tree/254a81ef488dcb3866cf8a4cacde501d9faaa588
```

Comment:
This one should be preserved since it's a different type

```
- Repo root: https://github.com/refined-github/sandbox/tree/254a81ef488dcb3866cf8a4cacde501d9faaa588
```